corrections by bfft

ebroecker opened the pull request:
Hi @BavariaBlue 

here is the next shot for my changes.

Could you please have a look on it and maybe merge?

best regards,
Eduard

ebroecker committed:
corrections by bfft

BavariaBlue commented on Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:
```diff
@@ -457,7 +457,7 @@ TODO change activity B of LSM1
 |:-------------|:------------- |:---------------|:-------------|
 |**Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released hardware and software versions and variants.** | **Minimize negative impacts for customers.**|
- **Activities** | **A.** Maintenance of attack pattern catalogue **B.** Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
+ **Activities** | **A.** Maintenance of attack pattern catalogue | **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
```
Activities: A. --> Where there is an A activity, there also must be a B Activity for LSM1.

ebroecker replied:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

BavariaBlue commented on Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:
```diff
@@ -703,18 +703,18 @@ TODO add levels & remove automotive-only character
-| ✦Are there project teams after final (serial) release responsible for these issues? | ??? | ??? | ??? | ??? |
+| ✦Does your organization gather information about new vulnerability issues? | NO | SOME | HALF  | MOST |
+| ✦Are these issues filtered to dedicated project, used hardware and software stacks? | NO | SOME | HALF  | MOST|
+| ✦Are there project teams after final (serial) release responsible for these issues? | NO | SOME | HALF  | MOST |
```
"NO SOME HALF MOST" is one option to answer. There are more options in the Assessment Excel sheet, as goes:
![image](https://user-images.githubusercontent.com/16705095/58404927-95d08100-8066-11e9-8039-c2d2f40f6236.png)

ebroecker replied:
I'll rethink these points

BavariaBlue commented on Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:
```diff
@@ -3171,9 +3171,9 @@ Since code signing is not appropriate for everything, architects and developers
-| **Activities** | **A.** Maintenance of attack pattern catalogue **B.** 1.	Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
-| **Assessment** | ✦Does your organization gather information about new vulnerability issues? ✦Are these issues filtered to dedicated project, used hardware and software stacks? ✦Are there project teams after final (serial) release responsible for these issues?| ✦Is your penetration test continuously developed further? ✦Do you assure, that penetration tests are executed for every released hardwar ...
-| **Results** | ✦Company has improved security knowledge ✦Up to date overview of potential attack surfaces ✦c.	Always up to date catalogue of attack patterns linked with probably impacts | ✦Detailed matrix which links attack patterns to ECUs ✦Database with references from software/hardware variants of ECU and possible attack patterns ✦Detailed TODO-List / Bug-List for each product/ECU. | ✦Company ...
+| **Activities** | **A.** Maintenance of attack pattern catalogue **B.** 1.	Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Operation of deployment infrastrucure **B.** Deploy  degradations/updates to cust ...
```
typo fix: infrastructure

ebroecker replied:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

BavariaBlue commented on Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:
```diff
@@ -3202,91 +3199,125 @@ TODO add long description + metrics, costs, personnel, related levels
-This leads also to many efforts to be made to keep this channel secure and still working.
-Security and reliability have to be in focus.
-After in step 2 (Doing) a security update has been developed or a potential degradation is decided this has to be deployed to the ECUs out in the field. Thereby only affected ECUs should get an update. This can be dependent to Hardware and or Software version and variant. Therefore, usually at OEM side a database with software and hardware Variants have to be maintained and used for the deploymen ...
+##### A. Operation of deployment infrastrucure
```
typo fix: infrastructure

ebroecker replied:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

BavariaBlue commented on Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:
```diff
@@ -3171,9 +3171,9 @@ Since code signing is not appropriate for everything, architects and developers
 | | **LSM1** | **LSM2** | **LSM3** |
 |:-------------|:-------------|:---------------|:-------------|:------------|
 | **Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released Hardware and Software versions and variants.** | **Minimize negative impacts for customers.**|
```
Hardware and Software --> hardware and software

ebroecker replied:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

BavariaBlue reviewed (changes requested):
Hi Eduard!

I added some comments to the pull request. Please update some of them if you feel it is necessary.

For future assessment questions, a long and short description is mandatory. Please provide these descriptions in this pull request. The location of these descriptions can be searched by the term "TODO add short" or "TODO add long" in SAMM-1.5.1.md.

Btw., great job by branching into BFFT and pull requesting into master back again 👍

ebroecker committed:
fix typos #13

this commit fixes the easy to fix typos.

the content remarks will follow later in another commit

ebroecker committed:
further typo fixes for #13

ebroecker commented:
@BavariaBlue 
Thanks for reviewing,
I hopefully fixed the easy to fix typos.

I'll rethink the questions/answers ~~and for sure I have to provide the  long and short description~~ the next days.

ebroecker committed:
add long description #13

ebroecker committed:
add short description #13
